SolomiiaLomnytska22/solomiia-training
# Adding "About" and "Contact" pages

**Refs:**
#11 (Add more pages)

**Proposed changes:**
- Added "About" page.
- Added "Contact" page.
- Added those pages to the router.ts.

User interface before implementing changes: 
https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/b23fc0ea-2069-4a3a-9a18-4d7f43548907

User interface after implementing changes:
<img width="1792" alt="Screenshot 2023-12-06 at 09 13 11" src="https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/e7e209fa-84fe-4794-8f9a-59449de8997f">
<img width="1791" alt="Screenshot 2023-12-06 at 09 13 42" src="https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/aaf6cffc-f675-42d1-a3b6-ab44789110cf">

**Verification:**
To see "About" page:
- Run the project.
- Click on the side menu icon.
- Select "About" page from the menu.

To see "Contact" page:
- Run the project.
- Click on the side menu icon.
- Select "Contact" page from the menu.

All changes tested locally. New and already existing functionality is supported.

https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/5876103e-7eee-498c-b04c-42908c59e42e
